qemu/tests/balloon_check.py

```diff
@@ -25,8 +25,8 @@ def __init__(self, test, params, env):
 
         self.vm = env.get_vm(params["main_vm"])
         self.session = self.get_session(self.vm)
-        logging.info("Waiting 90s for guest's applications up")
-        time.sleep(90)
+        logging.info("Waiting 180s for guest's applications up")
+        time.sleep(180)
```

Comment:
Is there a way to know 'applications up' rather than waiting longer?

Reply:
I did not find such a way.
Windows OS need some time to boot up itself, and autotest install and start some applications as well. 
Basically guest should be ready if its used mem keep stable. We can check guest used mem in a loop, when it's unchanged we mark it ready.
But this will also need a time period, such as we check the used mem in 3 minutes. So whatever method we use, they all take about 180s.
So I think the most fast and simple way is just extend the waiting time.

Reply:
No, it's not related to CVE bugs
As xuhan fix guest login time issue, now guest can login very fast, so 90s is not enough now.